public/data/kebab_data.json

```diff
@@ -5,48 +5,39 @@
         "address": "Rynek 20",
         "latitude": "51.20788328467099",
         "longitude": "16.161068016930532",
-        "place_id": "ChIJ9arvl5oSD0cRJm8AqJdF4LY",
+        "google_maps_url": "https://g.co/kgs/c3CxWnh",
```

Comment:
remove all "google mapls_url" fields, we don't have it in db and it stops seeder